docs/azure/TOC.yml

```diff
@@ -38,19 +38,19 @@
     - name: Authentication
       items:
       - name: Overview
-        href: sdk/authentication.md
+        href: /dotnet/azure/sdk/authentication/index
```

Comment:
Is this accurate @gewarren?

Reply:
It's accurate, just not in relative file format. For some reason the tool doesn't convert paths to relative in toc.yml files. I logged https://github.com/dotnet/docs-tools/issues/257.